research.md

```diff
@@ -1,88 +1,30 @@
-  - If you need any assistance:
-    - Talk to your DataRescue Guide if you are at an in-person event
-    - Or post questions on Slack in the #Researchers/Harvesters channel.
-    
-## Researchers and Harvesters
-- Researchers and Harvesters should work very closely together as their work will feed from each other and much communication is needed between the two roles.
-- For instance they could work in pairs or in small groups. 
-  - In some cases, a single person might be both a Researcher and a Harvester.
-- Note that in the Uncrawlable Action spreadsheet, Researchers and Harvesters share the same tab.
-- As a Researcher, make sure to check out the [Harvesters documentation](harvesting-toolkit) to familiarize yourself with their role.
+## Step 1: Claiming a dataset to research
 
-## Claiming a dataset to harvest
+As soon as they're seeded by the seeding group, URL's get added to the [Data rescue Archivers App](https://www.archivers.space). "Research" is the first phase of this process. Scroll or search through the list for a URL that seems appropriate (to your area of expertise, topical focus of the day's event, etc.). Make sure the URL has not been "checked out" by anyone else -- the `user` field should be blank.
 
-- Researchers work on datasets that were listed as uncrawlable by Seeders.
-- Go to the Uncrawlable spreadsheet, click the Researchers/Harvesters tab, and look for a dataset to harvest
-  - Available datasets are the ones whose cell "Researcher Handle" is empty
-  - If an item is already claimed but its "Date Opened or Closed" cell has turned red, it is also available for you to claim (for more details see the last section of this document)
-    - Claim it by entering your slack handle along with the status "Open" and today's date, for instance: 
-    ```
-    @khdelphine Open 1/22/2017
-    ```
-- You will find the URL you are about to evaluate in the "Original URL" cell.
+Click on the `URL` field to inspect the URL & learn enough about it to answer the questions in this section. Click on the `UUID` field to be brought to the app's record page for that URL, click the large blue `Checkout this URL` button, and begin working.
 
-## Evaluating the data
-Go to the URL, and start inspecting the content.
+## Step 2: Doing the research!
 
-## Is the data actually crawlable?
-Again, see [here](https://docs.google.com/document/d/1PeWefW2toThs-Pbw0CMv2us7wxQI0gRrP1LGuwMp_UQ/edit)
-and [here](https://docs.google.com/document/d/1qpuNCmBmu4KcsS_hE2srewcCiP4f9P5cCyDfHmsSAVU/edit)
-for a mostly non-technical introduction to the crawler. Some additional
-technical notes for answering this:
-- There is no specific file size cutoff on what is crawlable, but large files
-  should be manually captured anyway.
-- Files types like ZIP, PDF, Excel, etc. are crawlable if they are linked.
-- The crawler can only follow HTTP links that appear directly in the DOM at load
```

Comment:
These criteria are super important and I want us to have a reference that applies to EVERYONE and is handy in the repo. thoughts?